### ISSUE TITLE
xdp-filter: old man page LOAD option about "-s"

### ISSUE THREAD

**Issue by kp0zhiqian** [xdp-project/xdp-tools#53]
xdp-filter man page about LOAD has the old option `-s` while the` xdp-filter load --help` has the new one `-m`

```shell
root@netqe2 ~]# xdp-filter load -h

Usage: xdp-filter load [options] <ifname>

 Load xdp-filter on an interface

Required parameters:
  <ifname>                  Load on device <ifname>

Options:
 -F, --force                Force loading of XDP program
 -m, --mode <mode>          Load XDP program in <mode>; default native (valid values: native,skb,hw)
 -w, --whitelist            Use filters in whitelist mode (default blacklist)
 -f, --features <feats>     Features to enable; default all (valid values: tcp,udp,ipv6,ipv4,ethernet,all)
 -v, --verbose              Enable verbose logging (-vv: more verbose)
 -h, --help                 Show this help

[root@netqe2 ~]#
[root@netqe2 ~]# man xdp-filter | cat
(...)
The LOAD command
       To use xdp-filter, it must first be loaded unto an interface. This is accomplished with the load command, which takes the name of the interface as a parameter, and optionally allows  specify-
       ing  the  features  that  should  be  included.  By default all features are loaded, but de-selecting some features can speed up the packet matching, and increase performance by a substantial
       amount.

       The syntax for the load command is:

       xdp-filter load [options] <ifname>

       Where <ifname> is the name of the interface to load xdp-filter unto, and must be specified. The supported options are:

   -F, --force
       Specifying this option causes xdp-filter to unload any XDP program already loaded on the interface.

   -s, --skb-mode
       Specifying this option causes the XDP program to be loaded in the so-called skb mode (also known as generic XDP). This is a compatibility mode that results in lower performance,  but  can  be
       used on all network interfaces (whereas the default native XDP mode requires specific driver support).

```

**Comment by tohojo** [comment]
This was already fixed in 37f9d88aeaf1 and should be correct in 1.1.0-beta1.

**Comment by kp0zhiqian** [comment]
Got it~
